### ISSUE TITLE
年龄值和对应的概率值相乘再求和是什么意思

### ISSUE THREAD

**Issue by wangce888** [BoyuanJiang/Age-Gender-Estimate-TF#25]
年龄值和对应的概率值相乘再求和是什么意思
age_ = tf.cast(tf.constant([i for i in range(0, 101)]), tf.float32)
age = tf.reduce_sum(tf.multiply(tf.nn.softmax(age_logits), age_), axis=1)
为什么不能用argmax求最大可能性的年龄

**Comment by BoyuanJiang** [comment]
因为当时参考了一篇论文，它把对年龄的分类通过这种方式转换成了一个回归的问题
